src/python/tests/core/bot/fuzzers/syzkaller/syzkaller_fuzzer_test.py

```diff
@@ -42,3 +44,15 @@ def setUp(self):
     test_helpers.patch_environ(self)
 
     os.environ['BUILD_DIR'] = DATA_DIR
+
+
+class FuzzerTest(builtin_test.BaseEngineFuzzerTest):
```

Comment:
was there a reason this was added? this is testing libFuzzer and deprecated/outdated functionality at that.

Reply:
Ping on this comment.

Reply:
Woops, removed